src/form.js

```diff
@@ -187,3 +187,27 @@ export function isFieldDefault(field) {
   }
   return true;
 }
+
+/**
+ * Checks if a form field is empty.
+ * @param {!Element} field
+ * @throws {Error}
+ * @return {boolean}
+ */
+export function isFieldEmpty(field) {
+  switch (field.tagName) {
+    case 'INPUT':
+      if (field.type == 'checkbox' || field.type == 'radio') {
+        return !field.checked;
+      } else {
+        return !field.value;
+      }
+    case 'TEXTAREA':
+      return !field.value;
+    case 'SELECT':
+      // dropdown menu has at least one option always selected
+      return false;
+    default:
+      throw new Error(`${field.tagName} is not a supported field element.`);
```

Comment:
`${field.tagName} is not a supported field element to check emptiness?

Reply:
I appended to the method name to the message `isFieldEmpty: tag is not a supported field element`